File: 1.Two Sum/1.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int size=nums.size();
        vector<int> a;
        for(int i=0;i<size;i++)
            for(int j=i+1;j<size;j++)
            {
                if(nums[i]+nums[j]==target)
                {
                    a.push_back(i);
                    a.push_back(j);
                    return a;
                }
            }
    }
};

//Runtime: 84 ms
//Memory Usage: 1.4 MB
//Brute Force O(n^2)

File: 27. Remove Element/1.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        count=len(nums)
        i=0
        while i<count:
            if val==nums[i]:
                count=count-1
                nums.pop(i)
            else:
                i=i+1
        return count
       

#Runtime: 52 ms
#time waste:pop
#Memory Usage: 13.2 MB
